tests/end-to-end/external-solution-integration/pkg/testsuite/reset_counter_pod.go

```diff
@@ -0,0 +1,50 @@
+package testsuite
+
+import (
+	retrygo "github.com/avast/retry-go"
+	"github.com/pkg/errors"
+
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/retry"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/step"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/testkit"
+)
+
+// CheckCounterPod is a step which checks if counter has been updated in test pod
+type ResetCounterPod struct {
+	testService *testkit.TestService
+	retryOpts   []retrygo.Option
+}
+
+var _ step.Step = &ResetCounterPod{}
```

Comment:
What's the significance of this line?

Reply:
This is for compile time checking of full interface implementation adherence.